htmlui/src/Table.js

```diff
@@ -59,11 +59,12 @@ export default function MyTable({ columns, data }) {
     gotoPage,
     nextPage,
     previousPage,
-    state: { pageIndex },
+    setPageSize,
+    state: { pageIndex, pageSize },
   } = useTable({
     columns,
     data,
-    initialState: { pageSize: 10 },
+    initialState: { pageSize: 3 },
```

Comment:
10 perhaps?

Reply:
oops, this was put to three just for that last screenshot and i forgot to change it back.